msteams-platform/m365-apps/specify-runtime-requirements.md

```diff
@@ -0,0 +1,136 @@
+ms.date: 7/10/2024
+ms.author: mosdevdocs
+author: erikadoyle
+ms.topic: conceptual
+ms.localizationpriority: medium
+ms.subservice: m365apps
+---
+# Specify Microsoft 365 host runtime requirements in your app manifest
+
+> [!NOTE]
+>
+> The ability to specify Microsoft 365 host runtime requirements in app manifest is in [public developer preview](../resources/schema/manifest-schema-dev-preview.md).
+
+If you have a Microsoft Teams app that contains a personal tab or message extension, you can easily [extend it to run across additional Microsoft 365 hosts](./overview.md), including Microsoft Outlook and Microsoft 365 app. When you upgrade your Teams app to use app manifest version 1.13 or higher, your app is available in other Microsoft 365 application hosts by default. However, if your app also includes components that aren't yet supported across other Microsoft 365 hosts, your app might only partially load, resulting in unplanned and unsupported user experiences.
+
+To ensure your app always provides customers with high quality experiences, you can specify your app's runtime requirements in app manifest to tailor its behavior in applicable Microsoft 365 hosts, or omit it from surfacing in contexts that you're not ready to support. Describing your app's runtime requirements in app manifest also helps ensure your production-ready app experience reaches wider audiences in additional Microsoft 365 host applications in the future.
+
+Specifying your app's runtime requirements is useful in scenarios such as:
+
+- **One-way dependencies**: If your app includes both a tab and message extension, and the tab functions as a settings page for users to configure message extension functionality, you can ensure that your settings tab doesn't load in hosts that don't support your message extension.
+
+- **Mutual dependencies:** When your app has specific app components that must be loaded together in order to function properly, you can ensure your app is *only available* in Microsoft 365 hosts that support all components.  
+
+- **Capability requirements:** When your app has component parts with runtime requirements that aren't supported across all applicable Microsoft 365 hosts, you can ensure those components are *seamlessly omitted* (rather than loaded, yet not functional) from your app experience running in those hosts.
+
+## Microsoft 365 host support
+
+Only a subset of Microsoft 365 host applications supports the ability to specify runtime requirements in app manifest. This support will expand over time. The following hosts ensure only applicable apps and their applicable components are made available to users:
+
+|Microsoft 365 host application| Web | Desktop | Mobile |
+|---|---|---|---|
+| Teams| ✔️| ✔️| ✔️|
+| Outlook| ✔️| ✔️ (New Outlook only)| ✔️|
+| Microsoft 365 (Office)| | | |
+| Copilot for Microsoft 365 | | | |
+
+## Specify relationships between components of your app (`elementRelationshipSet`)
```

Comment:
```suggestion
## `elementRelationshipSet`
```

Reply:
I think we should include the explanatory description here, along with the API name. Seems more user friendly?

Reply:
User-friendly? Definitely. SEO-friendly? No. Search engines tend to pay less attention to headings after 60 characters.

Suggestion: _Specify relationships between components of your app_